recipes/openvino/all/conanfile.py

```diff
@@ -0,0 +1,402 @@
+        # SW plugins
+        "enable_auto": "Builds AUTO plugin",
+        "enable_hetero": "Builds HETERO plugin",
+        "enable_auto_batch": "Builds BATCH plugin",
+        # Frontends
+        "enable_ir_frontend": "Builds IR frontend",
+        "enable_onnx_frontend": "Builds ONNX frontend",
+        "enable_tf_frontend": "Builds TensorFlow frontend",
+        "enable_tf_lite_frontend": "Builds TensorFlow Lite frontend",
+        "enable_paddle_frontend": "Builds PaddlePaddle frontend",
+        "enable_pytorch_frontend": "Builds PyTorch frontend"
+    }
+
+    @property
+    def _protobuf_required(self):
+        return self.options.enable_tf_frontend or self.options.enable_onnx_frontend or self.options.enable_paddle_frontend
+
+    @property
+    def _target_arm(self):
+        return "arm" in self.settings.arch
+
+    @property
+    def _target_x86_64(self):
+        return self.settings.arch == "x86_64"
+
+    @property
+    def _gna_option_available(self):
+        return self.settings.os in ["Linux", "Windows"] and self._target_x86_64 and Version(self.version) < "2024.0.0"
+
+    @property
+    def _gpu_option_available(self):
+        return self.settings.os != "Macos" and self._target_x86_64
+
+    @property
+    def _preprocessing_available(self):
+        return Version(self.version) < "2023.2.0"
+
+    @property
+    def _onnx_version(self):
+        if Version(self.version) == "2023.1.0":
+            return "1.13.1"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "9",
+            "apple-clang": "11",
+            "Visual Studio": "16",
+            "msvc": "192",
+        }
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version]["openvino"], strip_root=True)
+        get(self, **self.conan_data["sources"][self.version]["onednn_cpu"], strip_root=True,
+            destination=f"{self.source_folder}/src/plugins/intel_cpu/thirdparty/onednn")
+        get(self, **self.conan_data["sources"][self.version]["mlas"], strip_root=True,
+            destination=f"{self.source_folder}/src/plugins/intel_cpu/thirdparty/mlas")
+        get(self, **self.conan_data["sources"][self.version]["arm_compute"], strip_root=True,
+            destination=f"{self.source_folder}/src/plugins/intel_cpu/thirdparty/ComputeLibrary")
+        get(self, **self.conan_data["sources"][self.version]["onednn_gpu"], strip_root=True,
+            destination=f"{self.source_folder}/src/plugins/intel_gpu/thirdparty/onednn_gpu")
+        apply_conandata_patches(self)
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if not self._gpu_option_available:
+            del self.options.enable_gpu
+
+    def configure(self):
+        suffix = "" if Version(conan_version).major < "2" else "/*"
```

Comment:
This syntax is valid in both 1.x and 2.0 so the extra logic is not required :)

Reply:
fixed